docs/framework/wpf/advanced/opentype-font-features.md

```diff
@@ -100,8 +97,8 @@ Text using OpenType capitals
 ### Titling Capitals  
  Titling capitals are lighter in weight and proportion and designed to give a more elegant look than normal capitals. Titling capitals are typically used in larger font sizes as headings. The following text displays normal and titling capitals for the Pescadero font. Notice the narrower stem widths of the text on the second line.  
   
- ![Text using OpenType titling capitals](../../../../docs/framework/wpf/advanced/media/opentypefont20.gif "OpenTypeFont20")  
-Text using OpenType titling capitals  
+ ![Text using OpenType titling capitals](./media/opentype-titling-capitals.gif "Text using OpenType titling capitals")  
```

Comment:
can you fix the warnings? All the paths have to be changed to contain the new folder. For example:

```suggestion
 ![Text using OpenType titling capitals](./media/opentype-font-features/opentype-titling-capitals.gif "Text using OpenType titling capitals")  
```

Reply:
Yes. Fixed. Please confirm that things look good.